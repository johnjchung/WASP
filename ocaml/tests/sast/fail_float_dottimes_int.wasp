Endpoint int_dottimes_float(GET:) {
	int x = 3.3 .* 4;
}
