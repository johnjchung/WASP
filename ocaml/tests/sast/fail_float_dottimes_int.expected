Fatal error: exception Type_checking.VariableDeclarationTypeError("Expected int expression, found float")
